imap_processing/tests/test_decom.py

```diff
@@ -0,0 +1,61 @@
+"""Tests aspects of the decommutation process that is generic to all instruments"""
+
+import glob
+
+import pytest
+
+from imap_processing import imap_module_directory
+
+xtce_documents = glob.glob(f"{imap_module_directory}/*/packet_definitions/*.xml")
```

Comment:
```suggestion
xtce_documents = imap_module_directory.glob("*/packet_definitions/*.xml")
```
imap_module_directory is a Path object so I think you can get rid of the glob import and use it directly on the Path instead.

Reply:
Oh cool! I didn't realize there was a `glob` method on `Path` objects.